main/unbound/root.hints

```diff
@@ -1,91 +0,0 @@
-;       This file holds the information on root name servers needed to
-;       initialize cache of Internet domain name servers
-;       (e.g. reference this file in the "cache  .  <file>"
-;       configuration file of BIND domain name servers).
-;
-;       This file is made available by InterNIC 
-;       under anonymous FTP as
-;           file                /domain/named.cache
-;           on server           FTP.INTERNIC.NET
-;       -OR-                    RS.INTERNIC.NET
-;
-;       last update:    August 25, 2016
-;       related version of root zone:   2016082500
```

Comment:
This file from a few years ago calls itself `named.cache` and presumably was being updated for some people by the cron job, but not being updated with `unbound` releases.